extensions/wikia/Search/classes/Hooks.php

```diff
@@ -40,7 +40,11 @@ class Hooks {
 	 * @return true
 	 */
 	public static function onArticleDeleteComplete( &$article, \User &$user, $reason, $id ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$user, $reason". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)